extension/content.js

```diff
@@ -113,6 +113,24 @@ function addTrendingMenuItem() {
 	`);
 }
 
+function addYoursMenuItem() {
+	const [, page] = location.pathname.split('/');
+	const username = getUsername();
+	const $menu = $('.subnav-links');
+
+	if ($(`.subnav-item[href="/${page}?q=is%3Aopen+is%3Aissue+user%3A${username}"]`).length > 0) {
+		return;
+	}
+
+	const yoursMenuItem = $(`<a href="/${page}?q=is%3Aopen+is%3Aissue+user%3A${username}" class="subnav-item">Yours</a>`);
```

Comment:
Passing down a string in two different spots seems convoluted compared to checking it inside the function.

This is what I see it changing to:
```
if (pageDetect.isIssueSearch() || pageDetect.isPRSearch()) {
	addYoursMenuItem(pageDetect.isIssueSearch() ? 'issues' : 'pulls');
}
```
Can you think of a cleaner solution than that? As that seems messier than what is existing.